### ISSUE TITLE
Submit application to Google for verification

### ISSUE THREAD

**Issue by xavierfoucrier** [xavierfoucrier/inbox-notifier#141]
Google has recently announced new policies for third-party access to Gmail APIs. Since the project is **used by Google Accounts outside an organization**, such as the general public: the application need to be submitted for verification by **February 15th, 2019**.

**Comment by xavierfoucrier** [comment]
Waiting for Google OAuth screen validation. :crossed_fingers:

**Comment by xavierfoucrier** [comment]
Still waiting for Google OAuth screen validation. :crossed_fingers:

**Comment by xavierfoucrier** [comment]
I got an answer from Google, and everything is OK on the application, except the main logo and application colors.

I will take the time to think and work on this part of the design for the v3.

**Comment by xavierfoucrier** [comment]
Working on `design-v3`.

**Comment by xavierfoucrier** [comment]
Waiting for Google OAuth screen validation for the second time. :crossed_fingers:
Everything in the version 3.0.0 should meet the Google OAuth requirements.

**Comment by xavierfoucrier** [comment]
Finally, Google say that **the current application name doesn't fit the OAuth requirements**... meaning that every references to "Gmail Notifier" official name should be reviewed and replaced.

I haven't take the decision, but renaming the app will take more time than expected.
Thinking for now :thinking: ...

**Comment by innuendo33** [comment]
hmm yes , it seems interesting but atm can't use it so useless :/

**Comment by xavierfoucrier** [comment]
Hi @innuendo33,

**Sorry for the time it takes**, but I haven't take a decision regarding this issue...
For now, you should be able to use the application by manually authorizing Gmail Notifier:

![gmn](https://user-images.githubusercontent.com/2471223/65590049-65eca800-df8a-11e9-8164-616bd3d9f3c7.png)

Please let me know if you are facing another issue. Thanks :+1: 

> Note that there is **a delay of 30 seconds** between the time you open the application, and the time you authorize Gmail Notifier to access your account: after this time, you need to retry the authorization process by closing the app and open it again.
>
> This is the **default behavior** of most applications working with Gmail services.

**Comment by innuendo33** [comment]
ok thanks anyway I won't use it then I want something flawless :)

**Comment by xavierfoucrier** [comment]
@innuendo33,

Sorry for the encountered problems.
The process of the application is flawless, but this is a particular situation because the application need to be verified by Google.

I need time to deal with that.

Thanks for your patience. :beers:

**Comment by xavierfoucrier** [comment]
Hi,

The OAuth screen authentication is now blocked by Google until I made the **appropriate branding changes** to the application: I have marked this issue as `critical`.

I am talking with the team to find a fast and clean solution.

**Comment by xavierfoucrier** [comment]
The current `v3.0.0` release has just been dropped from Github: every clients will **rollback to the previous version** automatically if the update service is still active on the application.

This will allow to properly choose a different name for the application, then patch the code, and finally validate the new OAuth screen with Google.

Thanks for your patience. :v:

**Comment by xavierfoucrier** [comment]
Hi there :hand: !

Just to let you know that I will work on the **v3** during December and want to drop a new version of the application before Christmas, I hope!

Stay tuned and thanks for your support :wink:

**Comment by xavierfoucrier** [comment]
Hi everyone :hand: !

Unfortunately I don't have much time to work on the project this week.
Big work will start next week. Again, I am really sorry for the delay.

Thanks for your support :wink:

**Comment by xavierfoucrier** [comment]
Hi everyone :hand: !

Just to let you know that **I have started working on the v3**.
See v3 branch here: https://github.com/xavierfoucrier/gmail-notifier/tree/v3

Again, sorry for the delay.

Hope to drop a new version as soon as possible!
Thanks for your support :wink:

**Comment by xavierfoucrier** [comment]
Waiting for Google OAuth screen validation for the last time, I hope! :crossed_fingers:

**Comment by xavierfoucrier** [comment]
Hi everyone,

As I have recently **replace the OAuth client secret code** to prepare the  v3 release, so you may encounter an error while trying to sync your inbox:

![02](https://user-images.githubusercontent.com/2471223/71766321-085be600-2eff-11ea-9370-f4815dcf7db5.png)

This will be fixed when the application will be Google certified: this may take one or two weeks, so be patient! :wink:

**Comment by xavierfoucrier** [comment]
Still waiting for Google certification. :crossed_fingers:

**Comment by xavierfoucrier** [comment]
Google team answer me yesterday, and there is still some fix to apply to the project, in addition to the other fix I made before. In most case, it's all about **privacy and project homepage**.

Thanks again for your patience. :wink: 

> Google is requiring **new modifications**, most time on each email answer, in the verification process, that's why it takes time to fix and re-apply to be certified...

**Comment by xavierfoucrier** [comment]
Hi everyone!

The application and privacy policy are now **Google compliant** :tada: 
> Next release **v3.0.0** will be published soon
